authLdap.php

```diff
@@ -254,7 +265,7 @@ function authLdap_get_server()
  * @param string $username
  * @param string $password
  * @param boolean $already_md5
- * @return boolean true, if login was successfull or false, if it wasn't
+ * @return WP_User, if login was successfull or false, if it wasn't
```

Comment:
Thanks 😂 

This should be `WP_Uer|WP_Error|false` to match the actual results, shouldn't it? ANd yes! It was definitely worse before! So: Thanks 😁

Reply:
I suppose it should, shouldn't it? 😂